llvm_jni/modules/jni.llvm/Context.bal

```diff
@@ -0,0 +1,62 @@
+import ballerina/jballerina.java;
+
+public distinct class Context {
+    handle LLVMContext;
+    public function init() {
+        initializeLLVM();
+        self.LLVMContext = create_llvm_context();
+    }
+
+    public function createBuilder() returns Builder {
+        return new (self);
+    }
+
+    public function createModule(string? name = ()) returns Module {
+        string modName = name ?: "";
+        return new (modName, self);
+    }
+}
+
+function initializeLLVM() {
+    llvm_link_initialize_core(llvm_get_global_pass_registry());
+    llvm_link_in_MCJIT();
```

Comment:
are we using llvm JIT features?

Reply:
This part was there just to validate code generated is correct by directly executing them. No longer required and removed.